### PR TITLE
Update 070-sales-channel-product-api.md

### PR DESCRIPTION
**Pull request by GerDner**
Add docs for nested associations and how to load them.

### 1. Why is this change necessary?

It was not clear to me how to load deep associations via storefront API.
I hope it saves some time for other devlopers who want to load more data in one request.

### 2. What does this change do, exactly?

Add docs/learnings from our Shopware-Client devlopment.

### CODE REVIEW

File: src/Docs/Resources/current/3-api/070-sales-channel-product-api.md [hunk 1]
```diff
@@ -20,5 +20,11 @@ Description: Returns detailed information about a specific product.
 
 **GET /sales-channel-api/v1/product/{productId}?associations[media][]**
 
+You can also load deep associations by providing multiple associations keys.
+
+**GET /sales-channel-api/v1/product/{productId}?associations[categories][associations][media][]**
+
+This will load the category assoassociation and the meda association of the category.
```
Comment (tinect) [inline src/Docs/Resources/current/3-api/070-sales-channel-product-api.md:27]: `assoassociation` => `association` 🤔

**Review by tinect — CHANGES_REQUESTED** [review]

File: src/Docs/Resources/current/3-api/070-sales-channel-product-api.md (hunk 1 above)
Comment (mitelg) [reply src/Docs/Resources/current/3-api/070-sales-channel-product-api.md:27]: the PR was already imported. we will fix that internally

Comment (GerDner) [reply src/Docs/Resources/current/3-api/070-sales-channel-product-api.md:27]: Sorry, was too fast on this one. Thx!

**Comment by mitelg** [comment]
thanks for your contribution @GerDner :+1: :tada: :blue_heart:

**Pull request closed without merging**